types/nhs-number-validator/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for nhs-number-validator 1.1
+// Project: https://github.com/spikeheap/nhs-number-validator
+// Definitions by: Josh Dando <https://github.com/Joshswooft>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function validate(nhsNumber: string | number): boolean;
```

Comment:
that's only half of the story, it probably has to support non-module script:
https://github.com/spikeheap/nhs-number-validator/blob/develop/index.js#L10
so it could be used like:
```ts
nhsNumberValidator.validate(...)
````
in global script
```suggestion
export function validate(nhsNumber: string | number): boolean;
export as namespace nhsNumberValidator;
```

Reply:
Thanks for the suggestion peter!